shm/player.cpp

```diff
@@ -0,0 +1,33 @@
+#include "player.hpp"
+
+#include <numeric>
+
+Player::Player(std::unique_ptr<Ship> ship, size_t money, size_t availableSpace)
```

Comment:
Please do not use size_t. Explanation on squad-7 discord channel.

Reply:
I replaced all size_t on uint_32t, now is good ?